src/backend/dx12/src/command.rs

```diff
@@ -38,8 +38,11 @@ fn get_rect(rect: &pso::Rect) -> d3d12::D3D12_RECT {
 }
 
 fn div(a: u32, b: u32) -> u32 {
-    assert_eq!(a % b, 0);
-    a / b
+    (a + b - 1) / b
```

Comment:
oh wow, this would change the semantics completely, wouldn't it?

Reply:
It's just `ceil` of `a / b`

Reply:
Heh, I misread. Indeed fine is it. /yoda